### PR TITLE
WX-1351 Speed up `Centaur Horicromtal PapiV2 Beta`

### PR DESCRIPTION
**Pull request by aednichols**
Before 1:34, now 1:17. Still the longest test, but now much less of an outlier compared to second-longest `Centaur Papi V2 Beta` at 1:09.

The gains were achieved by:
- Running more workflows in parallel
- Starting workflows faster

I'm continuing to focus exclusively on these two suites because the third-longest is TES at a mere 0:47.

### CODE REVIEW

**Comment by aednichols** [comment]
Example log output:

```
+---------------------+
| total_workflows_run |
+---------------------+
|                 382 |
+---------------------+
```

```
+----------------------------------------------+-----------------+----------------------------+----------------------------+
| name                                         | runtime_minutes | start                      | end                        |
+----------------------------------------------+-----------------+----------------------------+----------------------------+
| lots_of_inputs                               |              49 | 2023-12-04 19:11:09.757000 | 2023-12-04 20:00:16.994000 |
| localize_file_larger_than_disk_space         |              37 | 2023-12-04 19:06:22.131000 | 2023-12-04 19:44:00.567000 |
| call_cache_capoeira                          |              27 | 2023-12-04 19:06:42.671000 | 2023-12-04 19:34:22.956000 |
| draft3_call_cache_capoeira                   |              25 | 2023-12-04 19:17:55.395000 | 2023-12-04 19:43:15.246000 |
| papi_v2_plain_detritus                       |              21 | 2023-12-04 19:17:40.056000 | 2023-12-04 19:38:45.885000 |
| drs_usa_jdr                                  |              20 | 2023-12-04 19:17:33.937000 | 2023-12-04 19:37:43.666000 |
| invalid_runtime_attributes_wf                |              20 | 2023-12-04 19:16:21.027000 | 2023-12-04 19:36:30.736000 |
| google_labels                                |              18 | 2023-12-04 19:18:30.077000 | 2023-12-04 19:36:59.395000 |
| sub_workflow_interactions_scatter            |              18 | 2023-12-04 19:12:20.127000 | 2023-12-04 19:30:25.295000 |
| required_files                               |              16 | 2023-12-04 19:06:31.390000 | 2023-12-04 19:23:08.186000 |
| restart_while_failing                        |              16 | 2023-12-04 19:40:06.430000 | 2023-12-04 19:56:26.294000 |
| invalidate_bad_caches_no_copy                |              16 | 2023-12-04 19:12:11.857000 | 2023-12-04 19:28:40.915000 |
| cromwell_restart                             |              16 | 2023-12-04 19:11:25.197000 | 2023-12-04 19:27:37.625000 |
| custom_entrypoint_wf                         |              15 | 2023-12-04 19:06:15.944000 | 2023-12-04 19:21:24.272000 |
| write_lines_files                            |              15 | 2023-12-04 19:08:39.450000 | 2023-12-04 19:24:22.746000 |
| draft3_read_file_limits                      |              15 | 2023-12-04 19:05:25.828000 | 2023-12-04 19:20:38.471000 |
| sub_workflow_interactions                    |              15 | 2023-12-04 19:11:22.137000 | 2023-12-04 19:27:07.156000 |
| sizeenginefunction                           |              15 | 2023-12-04 19:16:20.006000 | 2023-12-04 19:31:31.987000 |
| two_step                                     |              15 | 2023-12-04 19:11:27.248000 | 2023-12-04 19:27:08.095000 |
| biscayne_read_functions_windows_line_endings |              14 | 2023-12-04 19:06:41.651000 | 2023-12-04 19:21:25.359000 |
+----------------------------------------------+-----------------+----------------------------+----------------------------+
```

File: src/ci/bin/test.inc.sh [hunk 1]
```diff
@@ -1607,6 +1607,42 @@ cromwell::build::generate_code_coverage() {
     fi
 }
 
+cromwell::build::print_workflow_statistics() {
+    echo "Total workflows"
```
Comment (THWiseman) [inline src/ci/bin/test.inc.sh:1611]: This is super cool

**Review by THWiseman — APPROVED** [review]

File: src/ci/bin/test.inc.sh (hunk 1 above)
Comment (jgainerdewar) [reply src/ci/bin/test.inc.sh:1611]: +1

**Review by jgainerdewar — APPROVED** [review]
Awesome stats!!!

**Pull request merged**